test/TestExtensions/OrleansTestingBase.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Orleans;
+using Orleans.Runtime;
+
+namespace UnitTests.Tester
+{
+    public abstract class OrleansTestingBase
+    {
+        protected static readonly Random random = new Random();
+
+        public Logger logger => GrainClient.Logger;
+
+        protected static IGrainFactory GrainFactory => GrainClient.GrainFactory;
+
+        protected static long GetRandomGrainId()
+        {
+            return random.Next();
+        }
+    }
+}
```

Comment:
because it was not generic, it was just used in a test (for no particular reason) but it's not logic that we want to put into this simplified project (and would brin unnecessary dependencies with it)